### PR TITLE
populated DEMISTO_SDK_NIGHTLY var to trigger test upload flow

### PR DESCRIPTION
**Pull request by yucohen**
## Description
We did not install the master sdk version in the test upload flow.
fixed it :) 

## Screenshots
image

### CODE REVIEW

File: Utils/trigger_test_upload_flow.sh [hunk 1]
```diff
@@ -151,4 +155,5 @@ curl --request POST \
   --form "variables[STORAGE_BASE_PATH]=${_storage_base_path}" \
   --form "variables[OVERRIDE_ALL_PACKS]=${_override_all_packs}" \
   --form "variables[CREATE_DEPENDENCIES_ZIP]=${_create_dependencies_zip}" \
+  --form "variables[DEMISTO_SDK_NIGHTLY]=${_demisto_sdk_nightly}" \
```
Comment (ilaner) [inline Utils/trigger_test_upload_flow.sh:158]:
```suggestion
  --form "variables[DEMISTO_SDK_NIGHTLY]=${DEMISTO_SDK_NIGHTLY}" \
```

File: Utils/trigger_test_upload_flow.sh [hunk 2]
```diff
@@ -86,6 +87,9 @@ while [[ "$#" -gt 0 ]]; do
   -o|--override-all-packs) _override_all_packs=true
     shift;;
 
+    -dsn|--sdk-nightly) _demisto_sdk_nightly=true
```
Comment (ilaner) [inline Utils/trigger_test_upload_flow.sh:90]:
```suggestion
```

File: Utils/trigger_test_upload_flow.sh [hunk 3]
```diff
@@ -15,6 +15,7 @@ if [ "$#" -lt "1" ]; then
   [-sbp, --storage-base-path] A path to copy from in this current upload, and to be used as a target destination. This path should look like upload-flow/builds/branch_name/build_number/content.
   [-dz, --create_dependencies_zip] Upload packs with dependencies zip
   [-o, --override_all_packs]  Whether to override all packs, and not just modified packs.
+  [-sn, --sdk-nightly]        Whether the flow is sdk nightly.
```
Comment (ilaner) [inline Utils/trigger_test_upload_flow.sh:18]:
```suggestion
```

File: Utils/trigger_test_upload_flow.sh (hunk 2 above)
Comment (yaakovpraisler) [reply Utils/trigger_test_upload_flow.sh]: @ilaner Why not give the option to set the SDK when triggering test upload?

**Review by yaakovpraisler — APPROVED** [review]

**Comment by yucohen** [comment]
needs for merge due to the sanity test issue in XSIAM - the rest passed

**Pull request merged**